gdx/src/com/badlogic/gdx/scenes/scene2d/ui/ParticleEffectActor.java

```diff
@@ -0,0 +1,100 @@
+package com.badlogic.gdx.scenes.scene2d.ui;
+
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.g2d.Batch;
+import com.badlogic.gdx.graphics.g2d.ParticleEffect;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+
+/**
+ * ParticleEffectActor holds an {@link ParticleEffect} to use in Scene2d applications.
+ * The particle effect is positioned in the centered in the ParticleEffectActor. Its bounding box
+ * is not limited to the size of this actor.
+ */
+public class ParticleEffectActor extends Actor {
```

Comment:
Maybe implement the disposable interface for the dispose method?

Reply:
Regarding the `dispose` implementation, I think it only makes sense in case it owns the `ParticleEffect`, that is if it's instantiated by the `ParticleEffectActor` (second and third constructors).

Reply:
It makes sense to call it if the ParticleEffect is just instantiated for this Actor and not referenced elsewhere, because some other ways to initialize it were needed. It should be called manually, and of course `getParticleEffect().dispose()` could used, too. I am open to remove it if others think it doesn't make sense here.

Reply:
Yes, like `Stage` `ownsBatch` we could have `ownsParticleEfffect` that would be `true` only when `ParticleEffect` is not provided but created internally.